tests/test_attribute_selector_block.py

```diff
@@ -49,3 +49,52 @@ def test_no_attributes_specified(self):
         self.assertDictEqual(
             self.last_notified[DEFAULT_TERMINAL][0].to_dict(),
             {'hello': 'n.io'})
+
+    def test_hidden_attributes(self):
```

Comment:
This looks good. I would probably add a test hidden for whitelist too, just for completeness